telegram/ext/_callbackcontext.py

```diff
@@ -389,6 +390,11 @@ def job_queue(self) -> Optional["JobQueue"]:
         .. seealso:: `Job Queue <https://github.com/python-telegram-bot/
             python-telegram-bot/wiki/Extensions-%E2%80%93-JobQueue>`_
         """
+        if self._application.job_queue is None:
+            warn(
+                "No `JobQueue` set up. To use `JobQueue`, you must install PTB via "
+                "`pip install python-telegram-bot[job_queue]`."
+            )
```

Comment:
the `if self._application.job_queue` will already trigger a warning if there is no `JobQueue`. But that warning will point the user to this line in `CallbackContext` instead of the line in the users code where they accessed `context.job_queue` … We could [surpress the first warning](https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings) and issue a new one (as currently done). but I'm not sure if it's worth the effort … cc @harshil21

Reply:
hmm or we could use the private attribute here instead of the property, so we bypass the warning. That's better isn't it?

Reply:
mh, would be okay with me, I guess. The only other way around that would be a `Application.get_job_queue` method, I guess and I still think that would be overkill